helper-cli/src/main/kotlin/commands/packagecuration/SplitCommand.kt

```diff
@@ -56,6 +56,8 @@ internal class SplitCommand : CliktCommand(
         .convert { it.absoluteFile.normalize() }
         .required()
 
+    private val idPattern = Regex("- id: \"(.+?)\"")
```

Comment:
consider using "".toRegex()

Reply:
While I'm personally not so much a fan of the `"".toRegex()` syntax, what would make sense here probably is to use triple-double-quotes (`"""`) to enclose the string to avoid the need to escape the contained `"`.